### ISSUE TITLE
CI broken because of gcc 4.9

### ISSUE THREAD

**Issue by danluu** [BitFunnel/NativeJIT#52]
https://travis-ci.org/BitFunnel/NativeJIT/jobs/141203696

I grabbed the CI setup from BitFunnel because that has a fix for a problem here. However, it also reverts gcc from gcc5 to gcc 4.9, which causes this failure:

~~~
/home/travis/build/BitFunnel/NativeJIT/src/CodeGen/FunctionBuffer.cpp: In member function ‘virtual void NativeJIT::FunctionBuffer::Reset()’:
/home/travis/build/BitFunnel/NativeJIT/src/CodeGen/FunctionBuffer.cpp:243:27: error: missing initializer for member ‘RUNTIME_FUNCTION::BeginAddress’ [-Werror=missing-field-initializers]
         m_runtimeFunction = {};
                           ^
/home/travis/build/BitFunnel/NativeJIT/src/CodeGen/FunctionBuffer.cpp:243:27: error: missing initializer for member ‘RUNTIME_FUNCTION::EndAddress’ [-Werror=missing-field-initializers]
/home/travis/build/BitFunnel/NativeJIT/src/CodeGen/FunctionBuffer.cpp:243:27: error: missing initializer for member ‘RUNTIME_FUNCTION::UnwindData’ [-Werror=missing-field-initializers]
~~~

I think Alex inserted some junk in BitFunnel to fix this gcc 4.9 issue in BitFunnel and that we can probably do something similar here.

**Comment by danluu** [comment]
From [this bug](https://gcc.gnu.org/bugzilla/show_bug.cgi?id=53119#c22), it looks like gcc doesn't care to backport fixes to stuff like struct initializer bugs to gcc 4.8/4.9.

**Comment by danluu** [comment]
I pushed a fix for this in 54e5c483f6260a3430657ec9ac11abbf1ceebf99 because it was trivial, but I don't think we should spend too much effort trying to maintain compatibility with older versions of gcc.
